src/style.css

```diff
@@ -0,0 +1,68 @@
+* {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-style: normal;
```

Comment:
dont use anything except box-sizing inside `*`, move font to html

Reply:
Thank you! Done!